packages/zod/src/v3/helpers/slug.ts

```diff
@@ -0,0 +1 @@
+export const isSlug = (value: string) => /^[a-z0-9]+(?:-[a-z0-9]+)*$/.test(value);
```

Comment:
You arent using this fn and zod doesnt have any other helper fns like this so ig this should be removed